recipes/install.rb

```diff
@@ -136,6 +136,39 @@
   only_if { node['cloud']['collect_logs'].casecmp?("true") && node['install']['cloud'].casecmp?("aws")}
 end
 
+bash 'Install pyenv' do
+  user 'root'
+  group 'root'
+  code <<-EOH
+    set -e
+    curl https://pyenv.run | bash
```

Comment:
we had an issue on load tests where it failed on this line since it didn't find PYENV_ROOT in the path 
https://github.com/logicalclocks/loadtest/pull/342/files#diff-4a2aea450d4b4160db6c83422647b1d2b59a0a04188d2f0ada9aeb49965ba107R5

Reply:
It shouldn't need the environment variable to install it. You need it only when you install/use a Python shim. I'll try in a new VM to verify

Reply:
actually i think it probably due to the cloud-init